src/otx/algorithms/anomaly/tasks/inference.py

```diff
@@ -359,7 +359,20 @@ def _add_metadata_to_ir(self, model_file: str, export_type: ExportType) -> None:
 
         extra_model_data[("model_info", "reverse_input_channels")] = False
         extra_model_data[("model_info", "model_type")] = "AnomalyDetection"
-        extra_model_data[("model_info", "labels")] = "Normal Anomaly"
+
+        labels = []
+        label_ids = []
+        for label_entity in self.task_environment.label_schema.get_labels(include_empty=False):
+            label_name = label_entity.name.replace(" ", "_")
+            # There is a mismatch between labels in OTX and modelAPI
```

Comment:
Can we rename the labels in MAPI later (perhaps that would be easier than modifying OTX)?